src/components/grid-aware/TwoParagraphBlock/TwoParagraphBlock.module.css

```diff
@@ -123,7 +124,6 @@
   }
 
   .image1Wrapper {
-    left: 40px;
     margin-bottom: 0;
     position: relative;
```

Comment:
Since you're not setting any of the left/top/right/bottom positions anymore, you don't need this `position: relative` either.

```suggestion
```

Reply:
Thanks for catching this!